src/client/components/lazy-chat/crisp.js

```diff
@@ -0,0 +1,27 @@
+import { crispWebsiteId } from '../../static/data/app.json'
+
+export const script = "https://client.crisp.chat/l.js"
```

Comment:
Use single quotes everywhere in this file? We don't use double quotes anywhere else I think.

Reply:
remnant of the old script, didn't notice it